master/internal/rm/kubernetesrm/pods.go

```diff
@@ -202,7 +204,11 @@ func Initialize(
 
 	err = p.startNodeInformer()
 	if err != nil {
-		panic(err)
+		if strings.Contains(err.Error(), "nodes is forbidden") {
+			ctx.Log().Warnf("unable to start node informer due to permission error: %s", err)
+		} else {
+			panic(err)
+		}
```

Comment:
nit: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow
so instead of this if/else flow, try 
```
if !string.Contains() or whatever other condition { 
     panic (err) 
}
p.syslog(...)
```

Reply:
Good call! I'll reorder these so it's clear, something along the lines of:
```
if err != nil && !errors.IsForbidden(err) {
    panic(err)
} else if err != nil {
    s.syslog(fmt.Sprintf("unable to start node informer due to permission error: %s", err)
}

Reply:
```
	switch {
	case err != nil && k8error.IsForbidden(err):
		p.syslog.Warnf("unable to start node informer due to permission error: %s", err)
	case err != nil:
		panic(err)
	}
```
really getting to personal preference here, but i prefer. (i prefer switch over if/else if/else for checking the results of a single call, and i prefer having the `k8error.IsForbidden` check not negated and next to the code that handles forbidden errors).

Reply:
Switch makes a lot more sense to me in this context, especially since this use is consistent with the rest of the file